frontend/src/app/main/data/workspace/drawing.cljs

```diff
@@ -42,6 +42,15 @@
           (when (= tool :path)
             (rx/of (start-drawing :path)))
 
+          (when (= tool :curve)
+            (let [stopper (->> stream (rx/filter dwc/interrupt?))]
+              (->> stream
+                   (rx/take-until stopper)
```

Comment:
The stopper should be the last position.

Reply:
ok